recipes/involucro/build.sh

```diff
@@ -1,5 +1,52 @@
-#!/bin/bash
+#! /bin/sh
 
-mkdir -p $PREFIX/bin
-cp ./involucr* $PREFIX/bin/involucro
-chmod +x $PREFIX/bin/involucro
+export GOPATH="$( pwd )"
+
+module_path="$( go env GOPATH )"/src/github.com/involucro/involucro/cmd/involucro
+GOBIN="$( go env GOBIN )" \
+  go get \
+  -ldflags="-s -X github.com/involucro/involucro/app.version=${PKG_VERSION}" \
```

Comment:
Upstream source has a placeholder `var version = "vDev"` which we fill out here.